codecov.yml

```diff
@@ -0,0 +1,26 @@
+coverage:
+  status:
+    project:
+      default: false
+      core:
+        target: 80%
+        flags:
+          - core
+      contrib:
+        target: 75%
+        flags:
+          - contrib
+    patch: yes
+
+flags:
+  core:
+    paths:
+      - ddtrace/*
+            - internal/*
+            - profiler/*
+  contrib:
+    paths:
+      - contrib/*
+
+
+comment: false
```

Comment:
Prevents Codecov PR comments

Reply:
But still there are comments in this PR, even a recent one, how come?

P.S. Add a new line here so that red icon goes away please.

Reply:
Should be good now! Looks like `comment: off` works